infra/images/podman-prod/configuration.nix

```diff
@@ -8,6 +8,7 @@
   imports =
     [ # Include the results of the hardware scan.
       ./hardware-configuration.nix
+      #./nginx.nix
```

Comment:
Commented out import

Reply:
That's intentional. We can't run this during the packer build because of the Let's Encrypt registration, but it's copying the files over to the image so they are ready. The Terraform is removing the comment and doing a rebuild to make it live one the real VM.

I'll add a clarifying comment to this file there.